tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -214,34 +214,39 @@ internal SecurityCoordinator(Security security, HttpContext context, Span span)
     /// <summary>
     /// Framework can do it all at once, but framework only unfortunately
     /// </summary>
-    internal void CheckAndBlock(Dictionary<string, object> args)
+    internal void CheckAndBlock(Dictionary<string, object> args, bool timeToTryReportSchema = false)
```

Comment:
nit: not a fan of the `timeTo` prefix 😅 
```suggestion
    internal void CheckAndBlock(Dictionary<string, object> args, bool tryToReportSchema = false)
```